pkgs/applications/science/logic/klee/default.nix

```diff
@@ -45,30 +45,24 @@ let
   };
 in stdenv.mkDerivation rec {
   pname = "klee";
-  version = "2.3";
+  version = "3.0";
 
   src = fetchFromGitHub {
     owner = "klee";
     repo = "klee";
     rev = "v${version}";
-    sha256 = "sha256-E1c6K6Q+LAWm342W8I00JI6+LMvqmULHZLkv9Kj5RmY=";
+    hash = "sha256-y5lWmtIcLAthQ0oHYQNd+ir75YaxHZR9Jgiz+ZUFQjY=";
   };
 
+  nativeBuildInputs = [ cmake ];
   buildInputs = [
     cryptominisat
     gperftools
-    lit # Configure phase checking for lit
```

Comment:
We should remove the `lit` function argument too (around line 16)

Reply:
It's still used in `nativeCheckInputs` below.

Reply:
Oops, somehow missed that. Sounds good.